### ISSUE TITLE
Update 1 (Our plan)

### ISSUE THREAD

**Issue by mnishikawa23** [stat231-s21/Blog-JAMS#2]
1. Do you plan for your final project to be an extension of the mid-semester project?

No. Our dataset for the shiny-app includes too few rows and leaves little space to expand our findings, explorations, and discussions. Thus, we have decided to move onto another topic.

Thus, we have modified our general interests to the following questions:

How does access to clean water differ across the world?
How many deaths are due to unsafe water sources / how does this differ across the world?
How does water access compare to life expectancy (across the world)?
Does water access differ by urban and rural regions?
How does water access compare to other socioeconomic factors?
What are the trends in people’s tweets seeking for a greater/more equitable access to water?

In order to address these questions, we are going to draw upon the following data sources:

Scraping and compiling of tweets through the twitter API
Downloading the files from the following URLs:

https://washdata.org/data/household#!/ (click download data)
https://ourworldindata.org/water-access (2017 data on share of deaths due to unsafe water; go to 2nd graph and click on download, the csv is called share-deaths-unsafe-water.csv)
https://www.sdg6data.org/ (the UN-Water Sustainable Development Goal 6 Data Portal)
https://ourworldindata.org/water-access (the fourth graph down shows the % of world population with access to improved drinking water in 2015)
https://data.worldbank.org/indicator (in general this is a good place for starting to look at various countries)

2. Our final product would include the visualization and descriptive analysis of the data acquired, which should illuminate the answers to the above raised questions by comparing the access to clean water across the world and across countries of different development levels. Although our visualization may include a map, we do not intend to develop an application with a significant amount of interactivity for this project.

3. Schedule:

Step 1: Sufficient Brainstorming about the detailed plan for the contents (decide what will be in each section/graphic on the blog and who will be responsible for completing each)
Deadline: By the end of this Sunday (written on 4/27/2021)

Step 2: Data Acquisition, Cleaning, and Wrangling
Deadline: Sunday, 5/2
Assign who prepares each data this weekend 
start and finish in the next week individually

Step 3: Data Analysis (Everyone code their section)
Deadline: depending on the date of the presentation, but hopefully before 5/10 or so

Step 4: Writing component of the blog 
(really good example = https://stat231-f20.github.io/Blog-Theory-of-Mind/ )
(Complete before our presentation date, so by 5/13 or 5/18)

The following two steps would take place between May 13th and 19th:

Step 5: Presentation

Step 6: Blog post completion/final edits

**Comment by katcorr** [comment]
@mnishikawa23 
@jcdailey21 
@AlastairPoole 
@sili22 

Sounds like a good plan, and an interesting blog post!  

(Also another solid blog from last semester, which may be helpful since they connected to the Twitter API and analyzed tweets, is [this one](https://stat231-f20.github.io/Blog-LetsGetFiscal/).)

Update 1: 10/10

**Comment by mnishikawa23** [comment]
Update 2 (some of other updates may follow):
we have already decided who should undertake what role, and therefore, we were able to initiate the process of data gathering and processing data. Thus, we have missed the internal deadline (5/2) in a sense but we already have a clear vision about how to finish up data acquisition. Thus, we are not particularly concerned about this delay. Also, Masahiro's data requires some time to be completed because he uses tweets and he wants to gather as many tweets as possible. He would again do the data acquisition process in Friday's morning and then move onto the data analysis and blog-writing part.

**Comment by jcdailey21** [comment]
Agreed with Masahiro's update. We've also each made a detailed plan of what visualizations our tabs will entail (which was a step we did a little too late to do in our last project), so I'm feeling good about where we're headed. 2 out of 4 of us are done with our data gathering/wrangling, and others should be by the end of the week. This will allow us to each code our sections this weekend as planned, and hopefully have most of the code done by this upcoming Monday (as we present on Thursday).

**Comment by sili22** [comment]
As Masahiro and Jamie mentioned, all of us have plans of what kind of dataset we will use for our tab, what visualization we are going to create, and what kind of data wrangling we need to perform. We planned to finish data-wrangling individually this week, and I will finish it tomorrow or latest on Thursday. After we are all done on Friday, we can proceed to code and write the blog.

**Comment by AlastairPoole** [comment]
I don't have any additional info to add- well said by my other group members. I'll just confirm what has already been said that we are well on pace to have the coding portion of the project done by this Monday.

**Comment by katcorr** [comment]
@mnishikawa23 @jcdailey21 @AlastairPoole @sili22 

Sounds good, glad you are on pace.  

Masahiro, to clarify -- you could start writing the analysis code based on your first scraping of the tweets.  That's a benefit of reproducible code -- it should still run when you go and scrape more tweets.  So, it's fine if you want to wait until Friday to scrape more then start the analysis code, but don't feel limited by needing *all* the "final" data before starting to write the analysis code.  Kind of like the calendar project, you can iterate between data collection and data wrangling/analysis so that you set yourself up with analysis code that should only need perhaps minor tweaking after you have all the data you want.  Does that make sense? 

Update 2: 5/5

**Comment by jcdailey21** [comment]
I second Masahiro's comments. Most of our coding contributions of the project (the hard part!) are complete, and now the primary focus is writing. I have most of the information about each persons' tabs to start writing a cohesive introduction and conclusion for the blog (please finish up this info tonight if you can so I can start writing tomorrow AM!) I am dedicating most of my day to this tomorrow to writing and aim to be finished with the majority of writing by 8 PM for my teammates to review, as we are also meeting then to also create/plan our presentation. There's a lot of writing and polishing to be done tomorrow, but I'm confident our blog will be in a great stage for our presentation Thursday.

**Comment by AlastairPoole** [comment]
I second what has been said and am confirming to make sure we are all on the same page. I finished up creating my tab (thank you for help during office hours Professor Correia!). I just need to polish up some of the labels which I am doing tonight. I will also then need to add it to our blog. Because I can clearly see what's happening in my tab, I am going to have my research questions, data sources, and conclusion also typed up tonight so that we can be prepared to discuss aspects of our presentation when we meet tomorrow.

**Comment by sili22** [comment]
I second my groupmates' comment. I finished writing both of my shiny apps and will add my own section of the blog post tonight so that Jamie will be able to write up the intro tomorrow. I look forward to discussing how we are going forward about our presentation at 8 pm.

**Comment by katcorr** [comment]
@mnishikawa23 @jcdailey21 @AlastairPoole @sili22 

sounds great, team!  in case it got buried among the various messages, I just want to highlight Jamie's comment so she has all the info needed to start writing tomorrow morning -- "I have most of the information about each persons' tabs to start writing a cohesive introduction and conclusion for the blog **(please finish up this info tonight if you can so I can start writing tomorrow AM!)**"

update 3: 5/5
